apps/smbmount.sh

```diff
@@ -19,6 +19,13 @@ root_check
 # Install cifs-utils
 install_if_not cifs-utils
 
+# Make sure, that name resolution works
+install_if_not winbind
+if [ "$(grep "^hosts:" /etc/nsswitch.conf | grep wins)" == "" ]
```

Comment:
Isn't it better to use `if [ -z .... ]` here?

Reply:
So that would be something like `if [ -z "$(grep "^hosts:" /etc/nsswitch.conf | grep wins)" ]`